src/components/Header/Header.js

```diff
@@ -26,27 +27,27 @@ const Header = props => (
       <img className={style.star} src={star} alt="star icon" />
       <span className={style.govAu}>GOV.AU</span>
       <span className={style.govWebsite}>Official Australian Government Website</span>
-      <div className={style.compactTitle}>Smart Cities</div>
-      <div className={style.compactSubtitle}>Performance Dashboard</div>
+      <div className={style.compactTitle}>National Cities Performance</div>
+      <div className={style.compactSubtitle}>Framework Dashboard</div>
```

Comment:
I assume this should be updated in the `<title>` of `index.html` as well? Anywhere else? The policy page stuff?

Reply:
I think I've caught them all, now ...